pkg/cmd/reset.go

```diff
@@ -17,11 +17,15 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/kubermatic/machine-controller/pkg/machines/v1alpha1"
```

Comment:
This import should go into a dedicated group below `k8c.io` imports. For example: https://github.com/kubermatic/kubeone/blob/0328596b6317603087b50ea8777816c0707bd039/pkg/templates/machinecontroller/machines.go#L19-L38